examples/rtspDeviceShifu/camera.py

```diff
@@ -8,6 +8,14 @@
 ip = os.environ.get("IP_CAMERA_ADDRESS")
 CAMERA_USERNAME = os.environ.get("IP_CAMERA_USERNAME")
 CAMERA_PASSWORD = os.environ.get("IP_CAMERA_PASSWORD")
+DEFAULT_SECRET_PATH = "/etc/edgedevice/secret/password"
+try:
+    with open(DEFAULT_SECRET_PATH, "r") as f:
+        CAMERA_PASSWORD = f.read()
+        print("Load password from secret.")
+except FileNotFoundError:
+    print("The secret password file does not exist.")
```

Comment:
show we throw an error or just print a log here?

Reply:
print a line of log means we solve this error in the program, if we throw this error out, it means this program can not solve the error and need the upper level to solve. shall we solve this error here?